invenio_app_ils/demo_data/documents.json

```diff
@@ -0,0 +1,183 @@
+[
+  {
+    "pid": "1",
+    "title": "An eventful journey to unification of all the fundamental forces",
+    "authors": [
+      {
+        "full_name": "Delbourgo, Robert",
+        "roles": ["Author"]
+      }
+    ],
+    "edition": "2",
+    "imprint": {
+      "date": "2020",
+      "place": "Hackensack, NJ",
+      "publisher": "World Scientific"
+    },
+    "abstract": "Book 2 of the Physics is arguably the best introduction to Aristotle's ideas, as well as being the most interesting and representative book in the whole of his corpus. It defines nature and distinguishes natural science from mathematics. It introduces the seminal idea of four causes, or four modes of explanation. It defines chance, but rejects a theory of chance and natural selection in favour of purpose in nature. Simplicius, writing in the sixth century AD, adds his own considerable contribution to this work. Seeing Aristotle's God as a Creator, he discusses how nature relates to soul, adds Stoic and Neoplatonist causes to Aristotle's list of four, and questions the likeness of cause to effect. He discusses missing a great evil or a great good by a hairsbreadth and considers whether animals act from reason or natural instinct. He also preserves a Posidonian discussion of mathematical astronomy.",
+    "languages": ["EN"],
+    "created_by": {
+      "type": "script",
+      "value": "demo"
+    },
+    "restricted": false,
+    "document_type": "BOOK",
+    "cover_metadata": {
+      "isbn": "9789811210143",
```

Comment:
for all of them, it is ok to add this here, but it is more important to add the ISBN in the `identifiers` field.